src/client/entity-editor/alias-editor/alias-editor.js

```diff
@@ -16,16 +16,18 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-import {Button, Col, Modal, Row} from 'react-bootstrap';
 
-import AddAliasButton from './add-alias-button';
-import AliasRowList from './alias-row-list';
+import {Button, Col, Modal, Row} from 'react-bootstrap';
+import {addAlias, hideAliasEditor} from './actions';
+import AliasRowContainer from './alias-row-container';
 import React from 'react';
 import {connect} from 'react-redux';
-import {hideAliasEditor} from '../actions';
+
 
 const AliasEditor = ({
+	aliases,
 	languageOptions,
+	onAddButtonClick,
```

Comment:
Would it make sense to name this as `onAddAlias`? We only care about the button click internally. Externally, we only care that some action has been performed to add an alias.